### ISSUE TITLE
multiple test set after learning

### ISSUE THREAD

**Issue by aphuongle95** [amaiya/ktrain#211]
Hi,
Is there a way to set the test set after the model has been trained?
Thank you,
Annie.

**Comment by amaiya** [comment]
Yes, you can run your dataset through the `preprocess_test` method of a `Preprocessor` instance.  Then, you can supply this processed test set as an argument to `learner.validate` (or `learner.evaluate`, which is an alias).  Alternatively, you can recreate the learner by calling `krain.get_learner` again using the same model and training set, but switching out `val_data` with the dataset processed by `preprocess_test`.

**Comment by aphuongle95** [comment]
Thank you :)
